[DOC] Add links to supported profile types in SDKs (#3228)
grafana/pyroscope: docs/sources/configure-client/grafana-agent/_index.md

- Versatile: eBPF can trace system calls, network packets, and even user-space application logic.
- Dynamic: No need to recompile or restart applications. eBPF allows for live tracing.

>  ### Set Up eBPF profiling with Grafana Agent:

Author: ```suggestion
### Set Up eBPF profiling with Grafana Agent:
```

- Centralized profiling: Suitable for environments with multiple Golang applications or microservices.
- Automatic: The agent handles the pulling and sending of profiles, requiring minimal configuration.

> ### Set Up Golang profiling in pull mode

Author: ```suggestion
### Set up Golang profiling in pull mode
```

1. Ensure your Golang application exposes pprof endpoints.
1. Install and configure the Grafana Agent on the same machine or container where your application runs.

> 1. Ensure the agent is set to pull mode and targeting the correct pprof endpoints. For step-by-step instructions, visit the [Go (Pull Mode)](/docs/pyroscope/latest/configure-client/grafana-agent/go_pull) docs.

Author: ```suggestion
1. Ensure the Agent is set to pull mode and targeting the correct pprof endpoints. For step-by-step instructions, visit the [Go (Pull Mode)](/docs/pyroscope/latest/configure-client/grafana-agent/go_pull) docs.
```

1. The Agent queries the pprof endpoints of your Golang application, collects the profiles, and forwards them to the Pyroscope server.

## Next steps

> Whether using eBPF for versatile system and application profiling or relying on Golang's built-in pprof endpoints in pull mode, Grafana Agent offers a streamlined process to gather essential profiling data.

Author: ```suggestion
Whether using eBPF for versatile system and application profiling or relying on Golang's built-in pprof endpoints in pull mode, Grafana Agent and Grafana Alloy offer streamlined processes to gather essential profiling data.
```
